hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/ClientInvocation.java

```diff
@@ -96,6 +96,7 @@ protected ClientInvocation(HazelcastClientInstanceImpl client,
                                int partitionId,
                                UUID uuid,
                                Connection connection) {
+        super(((ClientInvocationServiceImpl) client.getInvocationService()).invocationLogger);
```

Comment:
There is already a logger defined for client invocations. `this.logger = invocationService.invocationLogger;`. Can we get rid of it in favor of this?